### PR TITLE
review the new changes

### PR DESCRIPTION
**Pull request by vijayakumariPDMR**

### CODE REVIEW

File: src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -3,12 +3,12 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
-        //Second line
-        //Third line
+
         //Fourth line
         //fifth line
```
Comment (Anbuganeshpdmr) [inline src/main/java/org/example/Main.java]: also remove 5th line

**Review by Anbuganeshpdmr — CHANGES_REQUESTED** [review]
see the correction and make neccessarry change

File: src/main/java/org/example/Main.java (hunk 1 above)
Comment (vijayakumariPDMR) [reply src/main/java/org/example/Main.java]: done

**Review by vijayakumariPDMR — COMMENTED** [review]
recheck now

**Review by Anbuganeshpdmr — APPROVED** [review]
OK

**Pull request merged**